grommet/grommet
# typescript updates

#### What does this PR do?
Updates typescript autogenerated files

#### Where should the reviewer start?
any file changed
#### What testing has been done on this PR?
fresh clone of grommet core
#### How should this be manually tested?
`yarn test`
(fresh clone of grommet core, and run `yarn test` you'll see these file updated.)
#### Any background context you want to provide?
recently updated react-desc
#### What are the relevant issues?
none
#### Screenshots (if appropriate)
n/a
#### Do the grommet docs need to be updated?
no
#### Should this PR be mentioned in the release notes?
no
#### Is this change backwards compatible or is it a breaking change?
backwards compatible
